test/test_util/test_decomposition_util.py

```diff
@@ -113,7 +113,8 @@ def test_find_low_rank_throws_on_unknown_strategy() -> None:
         _ = find_low_rank(util, 2, util, cast(SVDStrategy, -5))
 
 
-def test_two_part_factor() -> None:
+def test_two_part_factor_rank_unknown() -> None:
+
```

Comment:
Looks like `black` wants to delete this line

Reply:
I removed the blank line with the latest commit - kind of odd that `black` did not complain about it on my system (black 24.1.1)

Reply:
Hmm--I was on 23.12, maybe they made some tweaks to the permitted styling?

Anyway, thanks & merging now!

Reply:
You're right! Introduced with version 24.1.0:

> Allow empty first lines at the beginning of most blocks (#3967, #4061)

[https://pypi.org/project/black/](https://pypi.org/project/black/)